ote_sdk/ote_sdk/entities/dataset_item.py

```diff
@@ -107,11 +114,28 @@ def __init__(
         if metadata is not None:
             self.__metadata = list(metadata)
 
+        self.__ignored_labels: Set[LabelEntity] = (
```

Comment:
Nice suggestion, but I prefer not to change it now (unless needed) because it could conflict with some type hints that are already in SC. Unfortunately `FrozenSet` is not a subtype of `Set`...